pkgs/by-name/wi/wifitui/package.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}: buildGoModule rec {
+  pname = "wifitui";
+  version = "unstable";
```

Is there any reason to not go with the current version (from last week)?
```suggestion
  version = "0.5.0";
```

---

I thght instead of going with fixed version numbers, going with unstable means it will be the latest always. I thght this helps me to not always be updated when the version changes.

Or am I wrong?